02-oop-inheritance/javaScript/refactored_JS/main  v0.1.js

```diff
@@ -0,0 +1,44 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function (paramName, paramValue) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
+  
+};//end movie constructor
+
+Movie.prototype = {
+
+  observer : new MovieObserver(), 
```

Comment:
This is not what I mean. I say you should add this to the new prototype object

constructor: Movie

The observer should go inside the constructor function